static/app/views/monitors/components/overviewTimeline/index.tsx

```diff
@@ -0,0 +1,109 @@
+import {Fragment, useCallback} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import Link from 'sentry/components/links/link';
+import Panel from 'sentry/components/panels/panel';
+import {SegmentedControl} from 'sentry/components/segmentedControl';
+import {IconSort} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import useRouter from 'sentry/utils/useRouter';
+
+import {Monitor} from '../../types';
+import {scheduleAsText} from '../../utils';
+
+interface Props {
+  monitorList: Monitor[];
+  monitorListPageLinks?: string | null;
+}
+
+export function OverviewTimeline({monitorList}: Props) {
+  const {replace, location} = useRouter();
+
+  const resolution = location.query?.resolution ?? '24hr';
+
+  const handleResolutionChange = useCallback(
+    (value: string) => {
+      replace({...location, query: {...location.query, resolution: value}});
+    },
+    [location, replace]
+  );
+
+  return (
+    <MonitorListPanel>
+      <ListFilters>
+        <Button size="xs" icon={<IconSort size="xs" />} aria-label={t('Reverse sort')} />
+        <SegmentedControl
+          value={resolution}
+          onChange={handleResolutionChange}
+          size="xs"
+          aria-label={t('Time Scale')}
+        >
+          <SegmentedControl.Item key="1hr">{t('Hour')}</SegmentedControl.Item>
+          <SegmentedControl.Item key="24hr">{t('Day')}</SegmentedControl.Item>
+          <SegmentedControl.Item key="7d">{t('Week')}</SegmentedControl.Item>
+          <SegmentedControl.Item key="30d">{t('Month')}</SegmentedControl.Item>
+        </SegmentedControl>
+      </ListFilters>
+      <TimelineScrubber />
+
+      {monitorList.map(monitor => (
+        <Fragment key={monitor.id}>
+          <MonitorDetails monitor={monitor} />
+          <TimelineContainer />
+        </Fragment>
+      ))}
+    </MonitorListPanel>
+  );
+}
+
+function MonitorDetails({monitor}: {monitor: Monitor}) {
+  const schedule = scheduleAsText(monitor.config);
+
+  return (
+    <DetailsContainer to="/crons/">
+      <Name>{monitor.name}</Name>
+      <Schedule>{schedule}</Schedule>
+    </DetailsContainer>
+  );
+}
+
+const MonitorListPanel = styled(Panel)`
+  display: grid;
+  grid-template-columns: 350px 1fr;
+`;
+
+const TimelineContainer = styled('div')`
+  border-bottom: 1px solid ${p => p.theme.innerBorder};
+`;
+
+const DetailsContainer = styled(Link)`
+  color: ${p => p.theme.textColor};
```

Comment:
For some reason I see some border-radius on this element, can we override? Not sure where this is coming from though :/

<img width="575" alt="image" src="https://github.com/getsentry/sentry/assets/9372512/280e6a66-7138-42c5-a7ee-e42509003d9d">